v3.0/how-to/upgrade/rolling-updates-with-ansible.md

```diff
@@ -18,18 +18,18 @@ When you perform a rolling update for a TiDB cluster, the service is shut down s
 - To upgrade between large versions, you must upgrade [`tidb-ansible`](https://github.com/pingcap/tidb-ansible).
 
     - To upgrade TiDB 2.0 (TiDB 2.0.1 or later) or TiDB 2.1 RC version to TiDB 2.1 GA version, see [TiDB 2.1 Upgrade Guide](/v2.1/how-to/upgrade/from-previous-version.md).
-    - To upgrade TiDB 2.0 (TiDB 2.0.1 or later) or TiDB 2.1 RC to TiDB 3.0, see [TiDB 3.0 Upgrade Guide](/how-to/upgrade/from-previous-version.md).
+    - To upgrade TiDB 2.0 (TiDB 2.0.1 or later) or TiDB 2.1 to TiDB 3.0, see [TiDB 3.0 Upgrade Guide](/how-to/upgrade/from-previous-version.md).
 
 - For a minor upgrade, it is also recommended to update `tidb-ansible` for the latest configuration file templates, features, and bug fixes.
 
 ### Download the binary automatically
 
 1. Edit the value of the `tidb_version` parameter in the `/home/tidb/tidb-ansible/inventory.ini` file, and specify the version number you need to upgrade to.
 
-    For example, to upgrade from `v2.0.6` to `v2.0.7`:
+    For example, to upgrade from `v3.0.0` to `v3.0.1`:
```

Comment:
Since `v3.0.1` hasn't been released, I think we'd better merge this PR later?

Reply:
Well, this PR touches a lot of different things, so I don't want to wait to merge the whole thing until 3.0.1 is released. If you want to defer this specific change, and maybe make a note to go back and add it when 3.0.1 is released, I guess that's OK, but it seems like having the upgrade instructions in place before 3.0.1 is released would be easiest and is not likely to cause any problems.